src/maps/components/AddressPicker.vue

```diff
@@ -192,7 +197,7 @@ export default {
 <style scoped lang="stylus">
 .map
   height 260px
-  margin-left 42px
+  margin-left 37px
```

Comment:
This makes it align nicely for place edit and user profile edit, but not group edit :/

Reply:
Seems fine to me...